### PR TITLE
chore: add 0.28.2 changelog

### PR DESCRIPTION
**Pull request by lihaosky**
### Description 
0.28.2 changelog

### Testing done 
NA

### Reviewer checklist
- [ ] Ensure docs are updated if necessary. (eg. if a user visible feature is being added or changed).
- [ ] Ensure relevant issues are linked (description should include text like "Fixes #<issue number>")

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,45 @@
...
+* change auth token provider to accept token strings instead of principals ([#9255](https://github.com/confluentinc/ksql/pull/9255)) ([b45841a](https://github.com/confluentinc/ksql/commit/b45841ac75ad7787ea64e23905e975a34ae7df56))
+* Excludes Guava from Guava-retrying in order to manage Guava dependencies more reliably. ([#9260](https://github.com/confluentinc/ksql/pull/9260)) ([c901ac4](https://github.com/confluentinc/ksql/commit/c901ac4f58e01ffab1a7e70f8261929e045abef8))
+* masked properties for create connector statement for 7.0.x ([#9266](https://github.com/confluentinc/ksql/pull/9266)) ([d471619](https://github.com/confluentinc/ksql/commit/d4716198040ac91f7c061650af4d238b7d08aac3))
+* masked properties for create connector statement for 7.1.x ([#9264](https://github.com/confluentinc/ksql/pull/9264)) ([0bd9c9c](https://github.com/confluentinc/ksql/commit/0bd9c9ce7c9524145bef619820f8eeba719f68de))
+* masked properties for create connector statement for 7.2.x ([#9263](https://github.com/confluentinc/ksql/pull/9263)) ([48a66f6](https://github.com/confluentinc/ksql/commit/48a66f6c82152fbdc3d6442ec851a252303d63df))
```
Comment (jnh5y) [inline CHANGELOG.md:33]: Does this need to be repeated?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,45 @@
...
+* change consumer_group_member_id tag to just member to match Druid name ([#9225](https://github.com/confluentinc/ksql/pull/9225)) ([904d995](https://github.com/confluentinc/ksql/commit/904d995f9b57e1ff059be364a3ca57f7b4912759))
+* respect authentication.skip.paths properly ([#9224](https://github.com/confluentinc/ksql/pull/9224)) ([4c33edd](https://github.com/confluentinc/ksql/commit/4c33eddafafda31d2b6046a8cf69621db45f2fb3))
+* remove topic tag conversion hack and update tests ([#9219](https://github.com/confluentinc/ksql/pull/9219)) ([adb7a67](https://github.com/confluentinc/ksql/commit/adb7a67e643f54e2d52cf81e36cc870025314de4))
+* masked properties for create connector statement for 5.5.x ([#9273](https://github.com/confluentinc/ksql/pull/9273)) ([2d81518](https://github.com/confluentinc/ksql/commit/2d815185b533bb9aa90ff343345f70e5fe8aa335))
+* masked properties for create connector statement for 5.4.x ([#9274](https://github.com/confluentinc/ksql/pull/9274)) ([23855ee](https://github.com/confluentinc/ksql/commit/23855eef5e7ec93e2698ecd7886ec46b19028bc5))
```
Comment (jnh5y) [inline CHANGELOG.md:40]: Same as above.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,4 +1,45 @@
...
+* classify KsqlSerializationException as USER error based on topic(KSE-1045) ([56dddbb](https://github.com/confluentinc/ksql/commit/56dddbb140ac90ee7658ad46b77ff252cd7f676c))
+* Create a KsqlSerializationException class ([e986f66](https://github.com/confluentinc/ksql/commit/e986f668e61bb58f83dfc927505fb509d35582e4))
+* DESCRIBE FUNCTION failing for annotated UDAFs with initial args ([#9277](https://github.com/confluentinc/ksql/pull/9277)) ([cc69c20](https://github.com/confluentinc/ksql/commit/cc69c2033a6e882948d2cc1f0603ab34b3ca965f))
+* masked properties for create connector statement for 6.0.x ([#9271](https://github.com/confluentinc/ksql/pull/9271)) ([0155c02](https://github.com/confluentinc/ksql/commit/0155c02b4f8818200c211c8b054c8d2fbf1a74bb))
+* masked properties for create connector statement for 6.1.x ([#9270](https://github.com/confluentinc/ksql/pull/9270)) ([e0868da](https://github.com/confluentinc/ksql/commit/e0868da99249a758d93026f9400ccd2e4fac1f44))
+* masked properties for create connector statement for 6.2.x ([#9269](https://github.com/confluentinc/ksql/pull/9269)) ([dae7e66](https://github.com/confluentinc/ksql/commit/dae7e66895626cf7554a3a6ad1cb2e496a19993c))
```
Comment (jnh5y) [inline CHANGELOG.md:24]: Ah, the cherry-picking + pint merge must have created a bunch of extra commits.

**Review by JimGalasyn — APPROVED** [review]
LGTM! I updated the 0.27.1 section from the 0.27.1-ksqldb branch.

File: docs/operate-and-deploy/changelog.md [hunk 4]
```diff
@@ -5,6 +5,10 @@ tagline: Detailed changelog for ksqlDB
 description: Lists changes to the ksqlDB codebase
 keywords: ksqldb, changelog
 ---
+Version 0.28.22
```
Comment (JimGalasyn) [inline docs/operate-and-deploy/changelog.md:8]:
```suggestion
Version 0.28.2
```

**Comment by raphaelauv** [comment]
0.28.2 is not on dockerhub -> https://hub.docker.com/r/confluentinc/ksqldb-server/tags

**Comment by lihaosky** [comment]
Thanks for asking @raphaelauv . We will upload it after further testing/verification.

**Pull request merged**